<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1232</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frame">
      <property name="minimumSize">
       <size>
        <width>718</width>
        <height>60</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>1</number>
       </property>
       <property name="topMargin">
        <number>1</number>
       </property>
       <property name="rightMargin">
        <number>1</number>
       </property>
       <property name="bottomMargin">
        <number>1</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="minimumSize">
          <size>
           <width>173</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>173</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="title_lab">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>0</y>
            <width>171</width>
            <height>51</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>171</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>173</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>VIG sms sender</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_3">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="label_status">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>10</y>
            <width>371</width>
            <height>31</height>
           </rect>
          </property>
          <property name="text">
           <string>status:</string>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frame_4">
      <property name="minimumSize">
       <size>
        <width>718</width>
        <height>0</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="spacing">
        <number>2</number>
       </property>
       <property name="leftMargin">
        <number>3</number>
       </property>
       <property name="topMargin">
        <number>3</number>
       </property>
       <property name="rightMargin">
        <number>3</number>
       </property>
       <property name="bottomMargin">
        <number>3</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stack_Window">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="send_window">
          <widget class="QLineEdit" name="line_phone">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>30</y>
             <width>481</width>
             <height>31</height>
            </rect>
           </property>
           <property name="text">
            <string>+79991112233</string>
           </property>
          </widget>
          <widget class="QLineEdit" name="line_message">
           <property name="geometry">
            <rect>
             <x>110</x>
             <y>100</y>
             <width>481</width>
             <height>151</height>
            </rect>
           </property>
           <property name="font">
            <font>
             <pointsize>10</pointsize>
            </font>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="alignment">
            <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_phone">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>30</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Phone</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
          <widget class="QLabel" name="label_message">
           <property name="geometry">
            <rect>
             <x>30</x>
             <y>100</y>
             <width>61</width>
             <height>31</height>
            </rect>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="text">
            <string>Message</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </widget>
         <widget class="QWidget" name="log_window">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QTextBrowser" name="text_logs">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_6">
         <property name="minimumSize">
          <size>
           <width>120</width>
           <height>390</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>120</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="leftMargin">
           <number>2</number>
          </property>
          <property name="topMargin">
           <number>2</number>
          </property>
          <property name="rightMargin">
           <number>2</number>
          </property>
          <property name="bottomMargin">
           <number>2</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_5">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>193</height>
             </size>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="start_but">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>71</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>Start</string>
             </property>
            </widget>
            <widget class="QPushButton" name="change_but">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>70</y>
               <width>71</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>Change</string>
             </property>
            </widget>
            <widget class="QPushButton" name="hide_but">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>130</y>
               <width>71</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>Hide</string>
             </property>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_7">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <widget class="QPushButton" name="stop_but">
             <property name="geometry">
              <rect>
               <x>20</x>
               <y>10</y>
               <width>71</width>
               <height>51</height>
              </rect>
             </property>
             <property name="text">
              <string>Stop</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
